### PR TITLE
Update Kubespray

### PR DESCRIPTION
**Pull request by NicolasT**

### CODE REVIEW

**Comment by NicolasT** [comment]
@bert-e wait

File: vendor/kubespray/roles/network_plugin/calico/defaults/main.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ nat_outgoing: true
 
 # add default ippool name
 calico_pool_name: "default-pool"
+calico_ipv4pool_ipip: "Off"
```
Comment (NicolasT) [inline vendor/kubespray/roles/network_plugin/calico/defaults/main.yml:7]: @Zempashi This looks fishy, no?

Comment (NicolasT) [reply vendor/kubespray/roles/network_plugin/calico/defaults/main.yml:7]: Ah no, default was `Off` anyway (https://docs.projectcalico.org/v3.3/reference/node/configuration and the env var used not to be set).

**Comment by NicolasT** [comment]
Nothing really sticks out, except:
- Update to K8s 1.12
- Updated version of Docker being deployed (18.06)
- Inclusion of a Docker 'orphan containers' cleanup script

**Review by Zempashi — APPROVED** [review]

**Pull request merged**